src/trio/_abc.py

```diff
@@ -192,9 +192,7 @@ async def getaddrinfo(
         Any required IDNA encoding is handled before calling this function;
         your implementation can assume that it will never see U-labels like
         ``"café.com"``, and only needs to handle A-labels like
-        ``b"xn--caf-dma.com"``.
-
-        """
+        ``b"xn--caf-dma.com"``."""  # spellchecker:disable-line
```

Comment:
xn--caf is both in extend-identifiers in pyproject.toml and has `spellchecker:disable-line`?

Reply:
For some reason extend-identifiers does does not work as I intended and continues to try replacing it, so I added `spellchecker:disable-line` there to solve the issue. If there would be a way to get extend-identifiers to do ignore that one like I intended, that would be a better solution.

Reply:
in that case we should at least remove the non-functioning line in extend-identifiers, unless you wanna report it upstream.